group-coordinator/src/main/java/org/apache/kafka/coordinator/group/consumer/ConsumerGroupMember.java

```diff
@@ -545,6 +547,32 @@ public String currentAssignmentSummary() {
             ')';
     }
 
+    public ConsumerGroupDescribeResponseData.Member asConsumerGroupDescribeMember(Assignment targetAssignment) {
+        return new ConsumerGroupDescribeResponseData.Member()
+            .setMemberEpoch(memberEpoch)
+            .setMemberId(Uuid.fromString(memberId))
```

Comment:
It may be better to change the member id field to a string as we use a string everywhere. I suppose that I got it wrong in the KIP.

Reply:
done